gems/sorbet/test/snapshot/run_one.sh

```diff
@@ -182,6 +182,7 @@ fi
   #
   # note: redirects stderr before the pipe
   info "├─ Running srb"
+  # shellcheck disable=SC2048
   if ! SRB_YES=1 ${test_cmd[*]} < /dev/null 2> "err.log" > "out.log"; then
```

Comment:
So tracing back the construction of `test_cmd` it does look like interpolation here could cause a problem if `$repo_root` contains spaces, since that goes into `$srb`. Does that sound right? If so, not sure it matters enough to fix it, but a quick comment might not hurt.

Reply:
These scripts are not used anymore (we re-implemented them using bazel) but i'm too sentimental to remove them.

Reply:
Ah right on :)